### DISCUSSION TITLE
Reaction Parameters

### DISCUSSION THREAD

**Discussion by caiosoas** [firemodels/fds#12932] [Q&A]
I use Pyrosim for a purpose of work and all simulations must use validated data. I can find some thermal parameters in the UMD or UL websites, but none of them gives me surface reaction parameteres (like HRRPUA and Ignition Temperature) nor material reaction parameteres (like Reference Temperature, Combustion Heat, etc). Where do you guys find those informations?

**Comment by mcgratta** [comment]
Ignition temperature typically comes from something like a cone calorimeter. HRRPUA is not something you find in a database. Rather, you find the HRR for a given burning item, like a chair, and you construct HRRPUA to match. Reference temperature is not something you would find in a database. That is something you would infer from TGA data.
